care/facility/api/viewsets/patient.py

```diff
@@ -147,8 +147,25 @@ def filter_by_category(self, queryset, name, value):
         field_name="last_consultation__symptoms_onset_date"
     )
     last_consultation_admitted_bed_type_list = MultiSelectFilter(
-        field_name="last_consultation__current_bed__bed__bed_type"
+        field_name="last_consultation__current_bed__bed__bed_type",
+        method="filter_by_bed_type",
     )
+
+    def filter_by_bed_type(self, queryset, name, value):
+        values = value.split(",")
+        union_queryset = PatientRegistration.objects.none()
+        for val in values:
+            val.strip()
+            if val == "8":
```

Comment:
Should we use `8` for filtering for no bed assigned?

What happens when we have to add a new bed type, but that new bed type's value would be 8 right? And this would cause conflict here, right?

```suggestion
            if val == "None":
```

Reply:
Got it!